pkg/pillar/types/zedroutertypes.go

```diff
@@ -1870,6 +1871,37 @@ type NetworkXObjectConfig struct {
 type IpRange struct {
 	Start net.IP
 	End   net.IP
+	Size  int
+}
+
+// Contains used to evaluate whether an IP address
+// is within the range
+func (ipRange IpRange) Contains(ipAddr net.IP) (bool, error) {
+	start := ipRange.Start.To4()
+	end := ipRange.End.To4()
+	cur := ipAddr.To4()
+	if cur == nil || start == nil || end == nil {
+		errStr := fmt.Sprintf("invalid configuration (%s, %s, %s)\n",
+			ipAddr.String(), ipRange.Start.String(), ipRange.End.String())
+		return false, errors.New(errStr)
+	}
+	if bytes.Compare(cur, start) >= 0 &&
+		bytes.Compare(cur, end) <= 0 {
```

Comment:
Question: Is the end part of the range or is it not? If the range is 128-255, then we shouldn't treat 255 as part of the range hence this should be < 0 and not <= 0

Reply:
I think your parseIpspec code to avoid using the broadcast address now means that <= 0 is the correct answer. Do you agree?